### PR TITLE
2D MRI extraction parameter file example

### PR DESCRIPTION
**Pull request by rcuocolo**
Created example parameter file with comments specific for 2D MRI extraction.

### CODE REVIEW

**Comment by fedorov** [comment]
Do you want to also add clarification to the documentation, as you suggested in the forum? The source files are here: https://github.com/Radiomics/pyradiomics/tree/master/docs.

**Comment by rcuocolo** [comment]
Thanks, I wasn't able to find the right file to edit. I will propose the
modification as soon as possible.

Renato

Il Lun 3 Feb 2020, 16:41 Andrey Fedorov <notifications@github.com> ha
scritto:

**Comment by JoostJM** [comment]
Related to issue #557

**Comment by rcuocolo** [comment]
Edited the documentation. I hope the options given by the resample and force2DDimension settings are more clearly presented.

File: docs/customization.rst [hunk 1]
```diff
@@ -153,8 +153,7 @@ Feature Extractor Level
 *Resampling the image*
 
 - ``resampledPixelSpacing`` [None]: List of 3 floats (>= 0), sets the size of the voxel in (x, y, z) plane when resampling.
-  A value of 0 is replaced with the spacing for that dimension as it is in the original (non-resampled) mask, thereby
-  enabling only in-plane resampling, for example.
+  A value of 0 is replaced with the spacing for that dimension as it is in the original (non-resampled) mask.For example, to perform only in-plane resampling, the x and y values alone should be edited (e.g.: [2,2,0]). In-plane resolution is always relative to image acquisition plane (i.e. axial, coronal or sagittal).
```
Comment (JoostJM) [inline docs/customization.rst]: Maybe a little nit picky, but can you add a space after the period?
`original (non-resampled) mask.For example, to`

**Comment by JoostJM** [comment]
The changes look good @rcuocolo! Thank you for your contribution. I only added a comment about one small typo. If you can fix that, I can merge this PR and after that, make a new release for PyRadiomics.

Don't worry about the failing tests, this was due to a bug I introduced in the test suite. I have fixed this today (in the current master: 2005bd2). It only affected the testing workflow. The regular workflow was not affected.

**Comment by rcuocolo** [comment]
Should be fixed now.

**Pull request merged**